.ci/setup.sh

```diff
@@ -17,7 +17,9 @@ if [[ $OS_NAME == "macos" ]]; then
             sudo xcode-select -s /Applications/Xcode_11.7.app/Contents/Developer || exit 1
         fi
     else  # gcc
-        sudo xcode-select -s /Applications/Xcode_14.1.app/Contents/Developer || exit 1
+        # Check https://github.com/actions/runner-images/tree/main/images/macos for available
+        # versions of Xcode
```

Comment:
forgot to say earlier... thanks very much for adding this comment on an otherwise kind of magical hard-coded string here 😁